### PR TITLE
Changes Discussed as an issue

### PR DESCRIPTION
**Pull request by acharmedmatrix**
Added Others dropdown option under theme in settings. Automatically defines how many tabs are needed. 
Added Sonarr calendar options in Sonarr module settings: default to week, start on sunday, M/D; added blue for unaired episodes, purple for airing now 
Added SABnzbd queue module

### CODE REVIEW

**Comment by acharmedmatrix** [comment]
So I'm not sure how to get PlexPy going, I'm getting an error from the API, but I see that you were involved in the creation of it. Any ideas on how to get this going?

**Comment by Hellowlol** [comment]
Thanks for the pr! Ill check ut out. Just a quick question is the code indented as it showed in on github or is this a simply a error from github?

Can you fix the code that quantified code is complaining about?

Get what going? What are you trying to do?

**Comment by acharmedmatrix** [comment]
Not sure on the indents, think it's a git glitch.  But I did a git clone of my branch and it ran well. 

Looks like all of the complaints are about the same line so might be an easy fix. I don't really know python too well so it's probably something redundant from me copying you. 

I'd like to do a PlexPy module or at the very least a Now Playing dashboard module.

**Comment by Hellowlol** [comment]
Ok, ill fix the code dont worry about it. As for nowplaying thats already included in htpc manager. Take a look at https://github.com/Hellowlol/HTPC-Manager/blob/master2/modules/plex.py#L888 and https://github.com/Hellowlol/HTPC-Manager/blob/master2/interfaces/default/js/plex.js#L646

**Comment by acharmedmatrix** [comment]
That's what I thought but it stopped working on me. Maybe it was a fluke. I'll look again.

**Comment by Hellowlol** [comment]
Let me know when your done, it did a quick test and the plex thing does not work, its a type so you can a undefined variable. You also need clean up the indentation etc. You can use a service like dirty markup for a linter for your editor.

**Comment by acharmedmatrix** [comment]
Yeah sorry, started working on the Plex thing this AM. 

I think it's good now. I have it working here with one stream, multiple streams, and no streams.

**Comment by Hellowlol** [comment]
As for the sabnzb h3 your can do something like

```
<h3>
    <div class="h3-sab">
        <div class="span4">link</div>
        <div class="span4 text-center">lol</div>
        <div class="span4 text-right">lol</div>
    </div>
</h3>
```

You need to remove all hardcorded width in the css, remember that htpc manager is suppose to fit all screens :)

```
<form action="" class="well form-inline disabled" id="add_nzb_form" method="post" name="add_nzb_form">
    <i class="rg rg-lg rg-sabnzbd-c"></i> <input class="span2" id="sab_pause_for" placeholder="time" type="number" value=""> <button class="btn" id="nzb_pause_button"><i class="fa fa-play"></i>
    Resume</button> <input class="span2" id="nzb_get_speed" placeholder="Kb/s" type="text" value=""> <button class="btn" id="nzb_set_speed"><i class="fa fa-arrow-circle-down fa-fw"></i> Set
    speed</button>
</form>
```

It would look like 
image

It still need some work but you get the idea.

Anyway flashing as to be fixed before i could merged it. its makeing me insane :P

**Comment by acharmedmatrix** [comment]
Solid points, for some dumb reason I was thinking the modules don't change width.

Yeah I don't love the flashing, I need to figure out how you do the refreshes on the module pages.

**Comment by Hellowlol** [comment]
you have to remove the content after the ajax call to stop the flashing.

**Comment by acharmedmatrix** [comment]
Not sure what you mean, aren't those thing relying on the Ajax? Or do you mean the setInterval?

File: interfaces/default/js/dash.js [hunk 1]
```diff
@@ -386,7 +430,34 @@ function loadRecentAlbumsPlex() {
     $('#albums-content-plex').parent().show()
   })
 }
-
+function loadCurrentPlex(){
+	  function CurrentPlexLoop(){
+  if (!$('#currentplex_table_body').length) return
+  $('#currentplex_table_body').empty()
```
Comment (Hellowlol) [inline interfaces/default/js/dash.js]: Move this line to line 438.

**Comment by acharmedmatrix** [comment]
I tried removing the setInterval like you did in sabnzbd.js, but that does not seem to have remedied it.

**Comment by acharmedmatrix** [comment]
Ok, I think I kinda figured it out... the empty() command seems to be causing the flash, but I'm not sure how to append the information where it overwrites rather than creating duplicate entries... working on it.

**Comment by Hellowlol** [comment]
Ok, just see what i did in https://github.com/Hellowlol/HTPC-Manager/commit/ebaae5ba268611e6b736afbdc20230e60e99d29c

**Comment by acharmedmatrix** [comment]
Funny I actually just got it before seeing your post, o well.

**Comment by acharmedmatrix** [comment]
Any idea on why I cannot execute scripts? They work perfectly from my terminal. Getting "/bin/sh: 1: test.sh: not found"

**Comment by Hellowlol** [comment]
Dunno works fine here

**Comment by acharmedmatrix** [comment]
Could you post your kill Plex script? Maybe I can see if that works for me and my script is the issue. 

I think I'm done with tweaking for a little bit. Let me know if you need/want any more changes for this.

**Comment by Hellowlol** [comment]
I can't remember is exactly but it's something like:

```
taskkill /f /im plexmediaserver.exe
```

**Comment by acharmedmatrix** [comment]
O maybe it's a linux issue then.

**Comment by Hellowlol** [comment]
So are you done with this so i can squash and lint your code? I isnt sure that all your code is gonna get merged, i might remove stuff i dont like.

**Comment by acharmedmatrix** [comment]
Yeah I think so. At some point I may try to make a PlexPy module but I think it's time for you to integrate. Let me know where everything ends up.

**Comment by Hellowlol** [comment]
i merged this just to get the ball rolling, ill clean it up later.

**Pull request merged**